### DISCUSSION TITLE
Is it possible to implement social logins?

### DISCUSSION THREAD

**Discussion by SSardorf** [fastapi-users/fastapi-users#629] [Q&A] [answered]
Anyone who can point me in a direction where I can implement social login in addition to login provided by fastapi-users?

**Comment by frankie567** [comment, marked as answer]
https://fastapi-users.github.io/fastapi-users/configuration/oauth/

**Reply by SSardorf** [reply]
Thanks a lot Frankie!

**Reply by fugacityengr** [reply]
Could the link in this answer be updated to a working link? Thanks

**Reply by frankie567** [reply]
https://fastapi-users.github.io/fastapi-users/configuration/oauth/
